lib/dialogs/rule.js

```diff
@@ -148,7 +148,7 @@ async function displayResult(dlg, app, { hasResult, echo }) {
             anyResult = true;
             if (count >= PAGE_SIZE) {
                 // ask the user before displaying further results
-                await dlg.replySpecial(dlg._("Show more result…"), 'more');
+                await dlg.replySpecial(dlg._("Show more results…"), 'more');
```

Comment:
Wait you kept the ... here, but dropped it everywhere else?

Reply:
Oops, forgot this one.